requirements.txt

```diff
@@ -1,62 +1,188 @@
+#
```

Comment:
If you blow away your `.venv` folder and create a new one, and then run `pip install -r requirements.txt` everything should hopefully work out the box